### DISCUSSION TITLE
Problemas na Instalação versão nova

### DISCUSSION THREAD

**Discussion by RodrigoChiarini** [RamonSilva20/mapos#1929] [Q&A]
Fiz como em um dos post anteriores, porem estou com problemas... alguma ideia do que seja?

![image](https://user-images.githubusercontent.com/115056832/193973164-98f9d60e-8213-43f4-983a-13eabdf4c529.png)

**Comment by vinicioaraujo** [comment]
Estou com o mesmo problema, alguma solução?

**Reply by vinicioaraujo** [reply]
@willph, por gentileza, poderia ajudar?

**Comment by SuperPorte** [comment]
se for instalação local no windows + xampp, eu criei um .bat que instala sem falhas (até agora)....

[Instalador-mapos-master - Copia.md](https://github.com/RamonSilva20/mapos/files/9821350/Instalador-mapos-master.-.Copia.md)

Precisei mudar a extensão de arquivo para .md pois o github não deixa enviar .bat
ALTERE o .md para .bat para conseguir usar o script.
Execute o script como ADM do windows para funcionar ok?

**Reply by vinicioaraujo** [reply]
Obrigado @bernarcom. Excelente iniciativa! Mas a instalação é web.

**Reply by vinicioaraujo** [reply]
Alguém tem uma solução?

**Reply by SuperPorte** [reply]
Conferiu a versão do php?
Eu SEMPRE deixo no php7.4

**Comment by marlontomaz** [comment]
Pessoal, bom dia! 

Esse erro é que a nova versão passou por updates de campos no banco de dados e a solução é primeiro baixar o arquivo completo do sistema, aqui do git e abrir a pasta Update na sua maquina local mesmo - > e abra o phpmyadmin do banco de dados la na hospedagem.... 
em seguinda, selecione o banco de dados que o sistema vai usar  e click em importar, e importe um a um cada update que esta disponivel nessa pasta
- na sequencia importe o seu backup e acesse a pagina de login
